src/WpfMath/TexFormulaParser.cs

```diff
@@ -487,10 +510,21 @@ private static SourceSpan ReadElement(SourceSpan value, ref int position)
             {
                 // Text style was found.
                 SkipWhiteSpace(value, ref position);
-                var styledFormula = this.Parse(ReadElement(value, ref position), command);
+
+                TexFormula styledFormula;
+                switch (command)
+                {
+                    case TexUtilities.TextStyleName:
+                        styledFormula = ParseChars(ReadElement(value, ref position), command);
+                        break;
+                    default:
+                        styledFormula = Parse(ReadElement(value, ref position), command);
+                        break;
+                }
+
```

Comment:
Are you sure we want `switch` here? Looks like it won't ever be expanded to cover any other commands? I'd use ternary expression.
```suggestion
var styledFormula = command == TexUtilities.TextStyleName
    ? ParseChars(ReadElement(value, ref position), command)
    : Parse(ReadElement(value, ref position), command);
```